tools/tpp-run/tpp-run.cpp

```diff
@@ -85,18 +85,26 @@ llvm::cl::opt<int> seed("seed",
                         llvm::cl::desc("Random seed, default 0 (no random)"),
                         llvm::cl::value_desc("int"), llvm::cl::init(0));
 
-// Speed optimization level
-llvm::cl::opt<unsigned>
-    optLevel("O", llvm::cl::desc("Speed optimization level (O0, O1, O2, O3)"),
-             llvm::cl::value_desc("0-3"), llvm::cl::init(2));
-
 // Initializer type
 // Default const if seed == 0, and normal otherwise
 llvm::cl::opt<std::string> initType(
     "init-type",
     llvm::cl::desc("Initializer type (const, simple, cont, rand, normal)"),
     llvm::cl::init(""));
 
+// Identity matrix
+// Replace single square argument with identity matrix
+// Note: Must have two arguments and the selected must be square
```

Comment:
I don't think it really follows any `a`/`b` "semantics".
Maybe the flag could just take `int` argument position directly?